Packages/vcs/vcs/unified1D.py

```diff
@@ -496,16 +510,21 @@ def _setlinecolor(self, value):
         self._linecolor = value
     linecolor = property(_getlinecolor, _setlinecolor)
 
-    def _getline(self):
-        return self._line
+    def _getlinetype(self):
+        return self._linetype
 
-    def _setline(self, value):
+    def _setlinetype(self, value):
         if value is not None:
-            if vcs.queries.isline(value):
-                value = value.name
-            value = VCS_validation_functions.checkLineType(self, 'line', value)
-        self._line = value
-    line = property(_getline, _setline)
```

Comment:
same here i think we should preserve backward compatibility for now and issue a deprecated warning